### PR TITLE
[amp-story-player] Add visual test for AMP version

### PR DESCRIPTION
**Pull request by Enriqe**
Closes https://github.com/ampproject/amphtml/issues/33562

### CODE REVIEW

**Comment by newmuis** [comment]
Can we also please add a manual QA test for this?

**Comment by Enriqe** [comment]
> Can we also please add a manual QA test for this?

Yep, for sure!

File: examples/visual-tests/amp-story-player/story-loaded-amp-version.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+'use strict';
+
+module.exports = {
+  'story loaded and AMP version player ready': async (page) => {
+    await page.waitForTimeout(5000);
+    await page.$eval('iframe', (iframe) => {
+      const contents = iframe.contentDocument.documentElement.outerHTML;
+      iframe.src = '';
+      iframe.srcdoc = contents;
+    });
```
Comment (gmajoulet) [inline examples/visual-tests/amp-story-player/story-loaded-amp-version.js:25]: Can you explain why this is needed?

Comment (Enriqe) [reply examples/visual-tests/amp-story-player/story-loaded-amp-version.js:25]: We found this is the only way Percy will render the story inside the iframe 🤷🏽‍♂️

File: test/visual-diff/visual-tests [hunk 2]
```diff
@@ -839,6 +839,14 @@
       "loading_incomplete_selectors": [".i-amphtml-story-player-loading"],
       "no_base_test": true
     },
+    {
+      "url": "examples/visual-tests/amp-story-player/player-amp-version.html",
+      "name": "amp-story-player AMP version",
+      "interactive_tests": "examples/visual-tests/amp-story-player/story-loaded-amp-version.js",
```
Comment (gmajoulet) [inline test/visual-diff/visual-tests:845]: Shouldn't the html and js share the same name?

**Review by gmajoulet — APPROVED** [review]

**Pull request merged**